pre_commit/languages/python.py

```diff
@@ -57,6 +58,9 @@ def install_environment(repo_cmd_runner, version='default'):
         repo_cmd_runner.run(venv_cmd)
         with in_env(repo_cmd_runner, version) as env:
             env.run("cd '{prefix}' && pip install .")
+            if additional_dependencies:
+                env.run("cd '{prefix}' && pip install -U" +
```

Comment:
My thinking is that it would allow for updating dependencies on subsequent runs, but I realized that without adding extra methods to verify the state of installed dependencies and run upgrades/additional installs then that wouldn't make sense anyway. In the current state, adding new dependencies requires `pre-commit clean` followed by a subsequent run to rebuild the environment.